modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerEnterpriseEditionTest.java

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.containers;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+import java.util.Collections;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.neo4j.driver.v1.AuthTokens;
+import org.neo4j.driver.v1.Driver;
+import org.neo4j.driver.v1.GraphDatabase;
+import org.neo4j.driver.v1.Session;
+
+public class Neo4jContainerEnterpriseEditionTest {
+
+    @ClassRule
+    public static Neo4jContainer neo4jContainer = new Neo4jContainer()
```

Comment:
Just as an info, you can also define the container inside a try-with-resources block (and manually call `start()`). This way, you could merge the test classes.

Not a requirement (since we don't follow a strong guideline here ourselves), just a remark.

Reply:
Thanks for the note, I wasn't sure what your preferred approach was.

I think I'd like to keep the enterprise test separate and merge the other two.